internal/app/dogma/util.go

```diff
@@ -268,3 +269,24 @@ func safeMarshalIndent(src []byte) []byte {
 	}
 	return src
 }
+
+type dogmaClientCtxKey struct{}
+
+var dogmaClientCtxKeyInstance = &dogmaClientCtxKey{}
+
+func getDogmaClientFrom(ctx context.Context) *centraldogma.Client {
```

Comment:
What do you think of creating `CommandContext` instead of storing `Client` to `cli.Context`? `CommandContext` may be created in the early phase of the CLI lifecycle.
```go
type CommandContext struct {
  *cli.Context
  *centraldogma.Client
}
```

Reply:
actually, using context to store and pass optional component is the common practice in Golang. I personally, am not against your method but it seems it's not the common practice. Should we keep it as it is, or any other strong reasons to go your way? @ikhoon

Reply:
The lifecycle of `centraldogma.Client` is the same as `cli.Context`, so I wanted to set it as a required value to access in the scope of command executions. `CommandContext` may make the code base cleaner and readable.

If we stil want to use `cli.Context` as it, I'm only with it. That said, what do you think of creating a Central Dogma client in one place and using `getDogmaClientFrom` in other places?